APCAppCore/APCAppCore/UI/Onboarding/EmailVerification/APCEmailVerifyViewController.m

```diff
@@ -467,6 +467,26 @@ - (IBAction) resendEmail: (id) __unused sender
         if (error != nil) {
             APCLogError2 (error);
         }
+        
+        dispatch_async(dispatch_get_main_queue(), ^{
+            NSString *message, *alertActionTitle;
+            
+            if (error != nil) {
+                message          = @"Unable to connect to server, please check your connection";
+                alertActionTitle = @"Retry";
+            } else {
+                message          = @"Verification email has been resent";
+                alertActionTitle = @"OK";
+            }
+            
+            NSString *localizedMessage          = NSLocalizedStringWithDefaultValue(message, @"APCAppCore", APCBundle(), message, @"");
+            NSString *localizedAlertActionTitle = NSLocalizedStringWithDefaultValue(alertActionTitle, @"APCAppCore", APCBundle(), alertActionTitle, @"");
```

Comment:
`NSLocalizedString` doesn't work this way. The script for pulling language changes to create the .Strings files requires that you have the string in the macro.